lib/features/reps_core/reps_core.module

```diff
@@ -268,6 +268,22 @@ function reps_core_block_view($delta = '') {
   return $block;
 }
 
+/**
+ * Implements hook_views_query_alter().
+ */
+ function reps_core_views_query_alter(&$view, &$query) {
+  
+ if ($view->name == 'reps_bean_blocks_global' && $view->current_display=='block_reps_bean') {
+
+   
+    $query->where[0]['conditions'][1]['field']='field_data_field_reps_core_node_reference.field_reps_core_node_reference_target_id';
```

Comment:
Do not hardcode positions in the array. Otherwise, walk through conditions with foreach loops:
```
foreach ($query->where as &$condition_group) {
      foreach ($condition_group['conditions'] as &$condition) {
          if ($condition['field'] == "field_data_field_reps_core_node_reference.field_reps_core_node_reference_target_id = :field_data_field_reps_core_node_reference_field_reps_core_node_reference_target_id"
   }
}
```

Reply:
ok